flask_rest_jsonapi/schema.py

```diff
@@ -106,7 +106,9 @@ def get_nested_fields(schema, model_field=False):
 
     nested_fields = []
     for (key, value) in schema._declared_fields.items():
-        if isinstance(value, List) and isinstance(value.container, Nested):
+        if isinstance(value, List) and isinstance(
```

Comment:
I tried it out in Badger without this fix, and defining field.List(field.string) still works :thinking:

Reply:
Discussed: I've only tried the getter functionality, which is fine. The problems start when we want to set this field